frontend/src/areas/AdminArea.js

```diff
@@ -21,11 +23,19 @@ export default function AdminArea({ children }) {
 	}, []);
 
 	if (!isLoaded){
-		return <></>;
+		return <CenterFlexBox>
+			<ParagraphTypography>
+				Loading
+			</ParagraphTypography>
+		</CenterFlexBox>;
 	}
 
 	if (!userIsAdmin){
-		return <></>;
+		return <CenterFlexBox>
+			<ParagraphTypography>
+				You are trying to access an admin restricted area. If you believe you should access to this area, please contact an admin directly.
```

Comment:
just a small typo :) You should _have_ access to this area

Reply:
Fixed.